src/libs/API/parameters/UpdateSageIntacctImportBooleanValues.ts

```diff
@@ -0,0 +1,6 @@
+type UpdateSageIntacctImportBooleanValues = {
```

Comment:
I think I would prefer we do what we've been doing with NetSuite so that we don't have to create all these type files:

https://github.com/Expensify/App/blob/2354386ed48e57a35d10e65cc149128264bcdd93/src/libs/API/types.ts#L529-L562

What do you think?